### DISCUSSION TITLE
Describe how to integrate Kratix and Backstage

### DISCUSSION THREAD

**Discussion by kaviankarimzadeh** [syntasso/kratix#47] [General]
Hi, 
The only document I've found is here, which doesn't provide many details about how integration should be implemented.

https://kratix.io/docs/main/how-kratix-complements/backstage#how

**Comment by kirederik** [comment]
hey kavian

sorry we missed this post :(

we added some further docs on how to integrate kratix and backstage here: https://docs.kratix.io/main/reference/backstage/intro

hope that helps!

closing it for now but feel free to reopen it :)
